coops-ui/src/components/AnnualReport/RegisteredOfficeAddress.vue

```diff
@@ -328,12 +328,11 @@ export default class RegisteredOfficeAddress extends Vue {
    */
   private sameAddress (address1: object, address2: object): boolean {
     const json1 = JSON.stringify(address1, (name: string, val: any) : any => {
-      return val !== '' && name !== 'actions' ? val : undefined
+      return val !== '' && name !== 'actions' && name !== 'addressType' ? val : undefined
```

Comment:
Please add parentheses around the 'if' part, to make this statement easier to understand :)

Reply:
Unit test will be a new ticket.